RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -525,6 +531,12 @@ boolean canEat(item toEat, boolean checkValidity)
 	{
 		return ($items[crappy brain, decent brain, good brain, boss brain, hunter brain, brains casserole, fricasseed brains, steel lasagna] contains toEat);
 	}
+	if(in_small())
+	{
+		// stomach size of 2 in small path
+		// small ignores level requirement so simply return here
```

Comment:
This isn't true. This check led to it pulling and attempting to eat a Tin Cup of Mulligan Stew, which is not possible.